src/utils/result-list.js

```diff
@@ -42,12 +42,16 @@ function chromeScrollList(elem: Element, index: number): void {
     return;
   }
 
-  const resultsHeight: number = resultsEl.clientHeight;
-  const itemHeight: number = resultsEl.children[0].clientHeight;
-  const numVisible: number = resultsHeight / itemHeight;
-  const positionsToScroll: number = index - numVisible + 1;
-  const itemOffset: number = resultsHeight % itemHeight;
-  const scroll: number = positionsToScroll * (itemHeight + 2) + itemOffset;
-
-  resultsEl.scrollTop = Math.max(0, scroll);
+  requestAnimationFrame(() => {
```

Comment:
See https://nolanlawson.com/2018/09/25/accurately-measuring-layout-on-the-web/ for some context.

Reply:
nice

Reply:
Lets turn this into a comment with that links so future readers will see it